### DISCUSSION TITLE
CORS/ERR_CONNECTION_RESET error when trying to upload larger files for a release

### DISCUSSION THREAD

**Discussion by theMK2k** [community/community#108087] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

While drafting a release and trying to upload a larger files (90MB - 170MB), I get the following error shown on the site:

![image](https://github.com/community/community/assets/16878526/bc943ede-e7ee-45b4-a033-c1eaed867762)

Checking the browser's console I get the following errors.

In Firefox:

> Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://objects-origin.githubusercontent.com/github-production-release-asset-2e65be. (Reason: CORS request did not succeed). Status code: (null).

In Chrome:

> POST https://objects-origin.githubusercontent.com/github-production-release-asset-2e65be net::ERR_CONNECTION_RESET

Tested with Firefox and Chrome and multiple internet service providers.

The error is sporadically so after numerous failed attempts I was able to upload 1 out of 16 files successfully.

Uploading small files (e.g. 100 bytes) works straight away.

Creating releases with similar sized binaries was not a problem in the past.

**Comment by jimsideout** [comment]
Same issue.

**Comment by theMK2k** [comment]
My issue still persists with the Web UI of github. However, when using the [Github CLI tool](https://cli.github.com/), it worked very well.
